### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by parvez-rashid**
Created the sign-up page. There are conflicts with the dev branch due to repeating index.html files, will fix these before merging.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+          id="input-text"
+          placeholder="Last name"
+          required
+        />
+
+        <label for="input-error">Password</label>
+        <input
+          type="pwd"
+          name="input-error"
+          class="txt-box error"
```
Comment (manish-GH) [inline index.html]: Don't use error here. The red border is unnecessary

File: index.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+          id="input-error"
+          placeholder="Passwords don't match"
+        />
+
+        <label for="input-error">Confirm password</label>
+        <input
+          type="pwd"
+          name="input-error"
+          class="txt-box error"
```
Comment (manish-GH) [inline index.html]: same here, don't use error

File: index.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+          id="input-error"
+          placeholder="Passwords don't match"
+        />
+
+        <button class="btn btn-primary" type="submit">Sign Up</button>
+        <span class="text-size-sm"
+          >Already have an account?
```
Comment (manish-GH) [inline index.html]: It might look better if this is centered

File: index.html [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+          <a class="options link" href="">wishlist</a>
+        </li>
+        <li>
+          <a class="options link" href="">cart</a>
+        </li>
+      </ul>
+    </nav>
+    <section class="signup-form">
+      <div class="input-container">
+        <h3 class="h3 text-wt-bold">Sign Up</h3>
```
Comment (manish-GH) [inline index.html]: It might look better if this is centered

**Review by manish-GH — APPROVED** [review]
The code is fine, there are no issues. The UI is clean and to the point. 

You can consider putting everything inside a card. The UI will look better.

File: index.html (hunk 1 above)
Comment (parvez-rashid) [reply index.html]: Was trying to show an error in password matching. But yes, I realise that it's not needed, will remove it.

File: index.html (hunk 3 above)
Comment (parvez-rashid) [reply index.html]: Sure, will do this.

**Review by Sahil25061999 — APPROVED** [review]
Same things as Manish suggested try putting the form inside the card rest looks good.

**Pull request merged**